composer.json

```diff
@@ -6,7 +6,7 @@
         "MIT"
     ],
     "require": {
-        "neos/flow": "^4.0 || ^5.0 || dev-master",
+        "neos/flow": ">=4.0, <= 6.0",
```

Comment:
Why is that considered better? It forbids `dev-master`, even when combined with `@dev`…